Community/service_point_watcher/README.md

```diff
@@ -1,23 +1,37 @@
 # Service Point Watcher  
```

Comment:
Ryu,
Each file, including READMEs, need the copyright and license template. Please add this at the top of each file and have style and formatting for the README after.

Copyright YYYY BlueCat Networks (USA) Inc. and its affiliates

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

By: Your Name (youremail@domain.com)
Date: DD-MM-YYYY
Gateway Version: X.X.X
Description: Brief description of what the workflow does and the expected behaviour

Reply:
Delme,
I Added / Revised the copyright and license template as per your instruction.
As for JSON files, I could not add since JSON files will see it as data.
For the README file, I commented it out as I did for the other files. 
Did you not want me to comment out the copyright and license paragraph on the README file?

Reply:
Ryu,
If you mean the previous format of the copyright and license, yes. No need to duplicate this info.
Thank you for the changes.

Reply:
Delme,
If you need me to change anything else please tell me, otherwise, I will wait for you to merge this to bluecatlabs.